projects/plugins/jetpack/_inc/client/components/settings-group/style.scss

```diff
@@ -25,7 +25,6 @@
 	.jp-form-setting-explanation {
 		color: $gray-text-min;
 		display: block;
-		margin: rem( 5px ) rem( 14px ) rem( 5px ) 0;
```

Comment:
Why this one? :-) The change is in a generic component used all over the place.

Reply:
Don't ask, fixed 😄